setup.py

```diff
@@ -402,7 +402,9 @@ def get_torch_version():
     version_number = version_number + local_version
 
 if wheel_name_suffix:
-    package_name = "{}-{}".format(package_name, wheel_name_suffix)
+    if not (enable_training and wheel_name_suffix == 'gpu'):
+        # for training packages, local version is used to indicate device types
+        package_name = "{}-{}".format(package_name, wheel_name_suffix)
```

Comment:
Thank you.

Reply:
Then the `wheel_name_suffix` is not used anywhere. Should the script exit if it was specified?

Reply:
You may also add such a test to https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/build.py#L1622

```python 
elif use_cuda and not ....
```